### PR TITLE
Build v3.7.8

### PR DESCRIPTION
**Pull request by marcoesters**
orjson, required by Anaconda Server (ticket [PKG-283](https://anaconda.atlassian.net/jira/software/c/projects/PKG/issues/PKG-283)).

# Package Info

Home: https://github.com/ijl/orjson
Development: https://github.com/ijl/orjson
Documentation: https://github.com/ijl/orjson

### CODE REVIEW

**Review by sumit0190 — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]
LGTM

Love this ↓
image

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,52 +1,42 @@
...
-    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-    - crossenv 1.0                        # [build_platform != target_platform]
-    - {{ compiler('c') }}                 # [not win]
-    - posix                               # [win]
+    - rust_win-64  # [win]
+    - {{ compiler('rust') }}  # [not win]
+    - {{ compiler('c') }}
     - cargo-bundle-licenses
   host:
+    - maturin >=0.13,<0.14
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:25]: It's better will be just `maturin 0.13`, see https://github.com/ijl/orjson/blob/3.7.8/CHANGELOG.md#changed

Comment (marcoesters) [reply recipe/meta.yaml:25]: That's what that line says, isn't it? See here: https://github.com/ijl/orjson/blob/3.7.8/pyproject.toml#L7

**Review by varlackc — COMMENTED** [review]
Maybe the following `doc_url` might be a easier for the user:
doc_url: https://github.com/ijl/orjson/blob/master/README.md

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**